docs/podman-container.1.md

```diff
@@ -20,7 +20,7 @@ The container command allows you to manage containers
 | create     | [podman-create(1)](podman-create.1.md)              | Create a new container.                                                      |
 | diff       | [podman-diff(1)](podman-diff.1.md)                  | Inspect changes on a container or image's filesystem.                        |
 | exec       | [podman-exec(1)](podman-exec.1.md)                  | Execute a command in a running container.                                    |
-| exists     | [podman-exists(1)](podman-container-exists.1.md)    | Check if a container exists in local storage                                 |
+| exists     | [podman-container-exists(1)](podman-container-exists.1.md)  | Check if a container exists in local storage                         |
```

Comment:
Aphabetize

Reply:
It already is, no? The useful sorting (the one the user cares about) is on the first column; this does not change that. Or have I misunderstood something?

Reply:
Yes sorry.